src/Spout/Writer/Common/Sheet.php

```diff
@@ -71,6 +75,23 @@ public function getName()
     }
 
     /**
+     * @api
+     * @return bool isVisible visibility of the sheet
+     */
+    public function isVisible()
+    {
+        return $this->isVisible;
+    }
+
+    public function setVisibility($visibility = true)
```

Comment:
Add PHPDoc. Change the signature to setIsVisible($isVisible) - name change and no default param.